app/src/main/java/com/darshan/notificity/Notification.kt

```diff
@@ -1,17 +1,15 @@
 package com.darshan.notificity
 
-import android.graphics.Bitmap
 import androidx.room.Entity
-import androidx.room.PrimaryKey
 
-@Entity
-data class NotificationEntity(
-    @PrimaryKey(autoGenerate = true) val id: Int = 0,
+@Entity(tableName = "notification", primaryKeys = ["id", "packageName"])
+data class Notification(
```

Comment:
@zzjjaayy Can you let the class name be the same? That way we save a lot of changes

Reply:
Makes sense. Changed back.